src/engine/input/key_mapping/InputEventItem.cs

```diff
@@ -292,6 +301,9 @@ private void OnRebindButtonPressed()
         WaitingForInput = true;
         button.Text = TranslationServer.Translate("PRESS_KEY_DOT_DOT_DOT");
         xButton.Visible = true;
+
+        // Signal to the input manager that a rebinding has started and it should ignore other input
```

Comment:
Re-reading this comment, shouldn't the "other" word be removed from here as the input manager is not part of the rebinding process and it ignores *all* input?